packages/ui/src/utils/utils.ts

```diff
@@ -0,0 +1,130 @@
+export const getInitials = (name: string, length?: number) => {
+  // Split the name into an array of words, ignoring empty strings
+  const words = name.split(' ').filter(Boolean)
+
+  // Get the initials from the words
+  const initials = words
+    .map(word => word[0].toUpperCase()) // Get the first letter of each word
+    .join('')
+
+  // If length is provided, truncate the initials to the desired length
+  return length ? initials.slice(0, length) : initials
```

Comment:
I wonder if this logic is enough. Product call, but if someone were to include their middle name, then I'd expect their 1st initial to be from their first name and 2nd to be from their last name (if the length was set to 2, which is probably pretty common).

So for _"Kevin Patrick Nagurski"_, I'd expect to see initials of _"KN"_, not _"KP"_.

One for another day I suppose.

Reply:
Yes, I think we’ll investigate this in more detail once we get to refactoring the component that uses this function.